### DISCUSSION TITLE
Do we need `asv`?

### DISCUSSION THREAD

**Discussion by natestemen** [unitaryfund/mitiq#1941] [General]
`asv` is a benchmarking tool that was added to mitiq last year [original infra [PR](https://github.com/unitaryfund/mitiq/pull/1047), deploying data to web [PR](https://github.com/unitaryfund/mitiq/pull/1354)]. Benchmarking data is stored in the [`mitiq-bench`](https://github.com/unitaryfund/mitiq-bench) repo, and displayed live at https://unitaryfund.github.io/mitiq-bench/. They are intended to track the performance of PEC + ZNE over time. That said, I'm afraid of two things:

1. They are too noisy to be useful. There are **huge** fluctuations in the error mitigation factor.
2. We aren't generally trying to improve the error mitigation performance of these techniques broadly over time.

As a result of these two, I don't think anyone is using the benchmarks. If this is the case, should we still have them?

**Comment by Misty-W** [comment]
My vote would be to remove `asv`.

**Comment by andreamari** [comment]
Fine to remove, also for me. Unless there is a way of making it more useful. But I don't see it.

**Comment by natestemen** [comment]
removed in https://github.com/unitaryfund/mitiq/pull/1960
